pkg/sippyserver/metrics/metrics.go

```diff
@@ -181,7 +181,7 @@ func refreshComponentReadinessMetrics(client *bqclient.Client) error {
 	log.Debugf("most recent GA is %q", mostRecentGA)
 	baseRelease := apitype.ComponentReportRequestReleaseOptions{
 		Release: mostRecentGA,
-		// Match what UI sends to API, although it's not correct TRT-1346
+		// Match what UI sends to API.
 		Start: releaseloader.GADateMap[mostRecentGA].AddDate(0, 0, -29),
 		End:   releaseloader.GADateMap[mostRecentGA].Add(-1 * time.Second),
```

Comment:
Hm, why didn't these values change? I would expect "4 weeks prior to GA" to be 28 days long, either ending at the start of the GA day, or at the end of the GA day.

Sippy stores the GA date as the start of the day, so in the case of end of GA day, it'd be something like this:

```
Start: releaseloader.GADateMap[mostRecentGA].AddDate(0, 0, -27),
End:   releaseloader.GADateMap[mostRecentGA].AddDate(0, 0, 1).Add(-1 * time.Second),
```

Does that make sense?

We need to make sure the values we're sending are accurate according to what the UI says we're sending, and that we're sending the same values when generating the metrics (so we make use of the cache)